images/server/install-packages.sh

```diff
@@ -2,26 +2,67 @@
 
 set -ex
 
+
+need_curl() {
+    if command -v curl >/dev/null ; then
+        return 0
+    fi
+    dnf install --setopt=install_weak_deps=False -y /usr/bin/curl
```

Comment:
Why do you need the `return 0`? maybe just have this one-liner:
```
command -v curl > /dev/null || dnf install -y --setopt=install_weak_deps=False curl
```

Reply:
Sure. I will keep it in mind as an optimization if I need to touch the script for any other reason.

Reply:
Fair enough.